configure.ac

```
@@ -3115,6 +3115,73 @@ else
   TCLTK_LIBS="$with_tcltk_libs"
 fi
 
+# check for _gdbmmodulec dependencies
+AC_CHECK_HEADERS([gdbm.h], [
+  LIBS_SAVE=$LIBS
+  AC_CHECK_LIB([gdbm], [gdbm_open])
+  LIBS=$LIBS_SAVE
+])
+
+# check for _dbmmodule.c dependencies
+AC_CHECK_HEADERS([ndbm.h], [
+  LIBS_SAVE="$LIBS"
+  AC_CHECK_LIB([ndbm], [dbm_open])
+  LIBS="$LIBS_SAVE"
+  AC_CHECK_LIB([gdbm_compat], [dbm_open])
```

I wonder if we can fold these into:

```suggestion
  AC_SEARCH_LIBS([dbm_open], [ndbm gdbm_compat])
```

---

``AC_SEARCH_LIBS()`` does not create ``HAVE_LIBFOO`` entries.

---

Ah, that's unfortunate.